tools/topology/topology1/development/sof-tgl-nocodec-ci.m4

```diff
@@ -122,7 +122,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-demux-playback.m4,
 
 # Passthrough capture pipeline 2 on PCM 0 using max 2 channels of s16le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
```

Comment:
we use pipe-volume-switch-capture, can you please align?

Reply:
> we use pipe-volume-switch-capture, can you please align?
hi @plbossart , keyon is on vacation . Would you mind checking this 
 one https://github.com/thesofproject/sof/pull/4463 ?